paasta_tools/metrics/metastatus_lib.py

```diff
@@ -215,7 +215,7 @@ def assert_tasks_running(metrics):
     )
 
 
-def assert_no_duplicate_frameworks(state):
+def assert_no_duplicate_frameworks(state, ignore=[]):
```

Comment:
I'd actually just make this ``assert_no_duplicate_frameworks(state, framework_name`` and call it for marathon + chronos, rather than the inverse